src/Simulation/Simulators.Tests/TestProjects/HoneywellExe/ClassicallyControlledSupportTests.qs

```diff
@@ -23,724 +23,658 @@ namespace Microsoft.Quantum.Simulation.Testing.Honeywell.ClassicallyControlledSu
             }
             Reset(q);
         }
-    }
-
-    operation BasicLift() : Unit {
-        let r = Zero;
-        if (r == Zero) {
-            SubOp1();
-            SubOp2();
-            SubOp3();
-            let temp = 4;
-            using (q = Qubit()) {
-                let temp2 = q;
-            }
-        }
-    }
-
-    operation LiftLoops() : Unit {
-        let r = Zero;
-        if (r == Zero) {
-            for (index in 0 .. 3) {
-                let temp = index;
-            }
-
-            repeat {
-                let success = true;
-            } until (success)
-            fixup {
-                let temp2 = 0;
-            }
-        }
-    }
-    
-    operation LiftSingleNonCall() : Unit {
-        let r = Zero;
-        if (r == Zero) {
-            let temp = 2;
-        }
-    }
-    
-    operation DontLiftReturnStatements() : Unit {
-        let r = Zero;
-        if (r == Zero) {
-            SubOp1();
-            return ();
-        }
-    }
-
-    function SubFunc1() : Unit { }
-    function SubFunc2() : Unit { }
-    function SubFunc3() : Unit { }
-
-    function DontLiftFunctions() : Unit {
```

Comment:
Why is this one removed? This should be valid code.

Reply:
This code generates the following error:
`QS5024: Measurement results cannot be compared here. The execution target HoneywellProcessor only supports comparing measurement results as part of the condition of an if- or elif-statement in an operation.`
This error seems appropriate and tells me this code is not appropriate as an execution test.

Reply:
Ah, I see - DontLiftFunctions itself is a function.